### ISSUE TITLE
CUDA_ERROR_LAUNCH_FAILED when training RNNs

### ISSUE THREAD

**Issue by AndreasPangerl** [tensorflow/tensorflow#39267]
I'm getting the following error when training LSTMs or GRUs

I tensorflow/stream_executor/stream.cc:1868 [stream=0000019AB6A744F0,impl=0000019ACE30D210] did not wait for [stream=0000019AB6A73AF0,impl=0000019ACE30D4B0]
I tensorflow/stream_executor/stream.cc:4816 [stream=0000019AB6A744F0,impl=0000019ACE30D210] did not memcpy host-to-device; source: 0000019F7DB15080
I tensorflow/stream_executor/stream.cc:1868] [stream=0000019AB6A744F0,impl=0000019ACE30D210] did not wait for [stream=0000019AB6A73AF0,impl=0000019ACE30D4B0]
I tensorflow/stream_executor/stream.cc:4816] [stream=0000019AB6A744F0,impl=0000019ACE30D210] did not memcpy host-to-device; source: 0000019A943627C0
E tensorflow/stream_executor/stream.cc:332] Error recording event in stream: error recording CUDA event on stream 0x19ad7cd0600: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure; not marking stream as bad, as the Event object may be at fault. Monitor for further errors.
E tensorflow/stream_executor/cuda/cuda_event.cc:29] Error polling for event status: failed to query event: CUDA_ERROR_LAUNCH_FAILED: unspecified launch failure
F tensorflow/core/common_runtime/gpu/gpu_event_mgr.cc:273] Unexpected Event status: 1
[I 13:52:21.664 LabApp] KernelRestarter: restarting kernel (1/5), keep random ports

The problem occurs only on the newly bought machine, not on the previously used one, although both have the same setup with same tensorflow and CUDA versions:

Intel Core i9 9960X
Nvidia Quadro RTX 6000
Windows 10
Tensorflow 2.0
CUDA 10.1
cuDNN 7.6.5

Also, all users properly initialize the GPU in the jupyter notebooks to allow for multiple notebooks being trained with

gpus = tf.config.experimental.list_physical_devices('GPU')
tf.config.experimental.set_memory_growth(enable=True,device=gpus[0])

I would appreciate any help, since I haven't found any solution for this on the web.

Thanks!

**Comment by amahendrakar** [comment]
@AndreasPangerl,
In order to expedite the trouble-shooting process, could you please provide the complete code to reproduce the issue reported here. Thanks!

**Comment by AndreasPangerl** [comment]
def create_model(stateful, num_lstm, size_lstm, num_dense, size_dense, batch_size, optimizer):
    
    model = Sequential()
    model.add(Masking(mask_value=0.0))

    model.add(Bidirectional(merge_mode='ave',layer=LSTM(200,
                  input_shape=X_train.shape[1:3],
                  batch_size=batch_size,
                  stateful=stateful,
                  return_sequences=True),
                  batch_input_shape=(batch_size, timeSteps, numFeats)))

    model.add(Bidirectional(merge_mode='ave',layer=LSTM(100,
                  input_shape=X_train.shape[1:3],
                  batch_size=batch_size,
                  stateful=stateful,
                  return_sequences=True),
                  batch_input_shape=(batch_size, timeSteps, numFeats)))

    model.add(Dense(100,activation='selu',kernel_initializer=lecun_normal()))
    model.add(Dense(50,activation='selu',kernel_initializer=lecun_normal()))
    model.add(Dense(25,activation='selu',kernel_initializer=lecun_normal()))
    model.add(Dense(1,activation="linear"))
  
    model.compile(loss=losses.mean_squared_error, optimizer=optimizer)
    return model

    model_stateful =create_model(stateful=False,optimizer=Adam(lr=0.001),batch_size=200

    ModelCpt = ModelCheckpoint("model_1.h5", monitor="loss", save_best_only=True)
    ReduceLRCpt = ReduceLROnPlateau(monitor='loss', factor=0.85, patience=5, min_lr=1e-7,verbose=1)

    history_stateful = model_stateful.fit(X_train,
                                      Y_train,
                                      batch_size=batch_size_aux,epochs=2500,verbose=1,
                                      validation_data=(X_test, Y_test),
                                      callbacks=[ModelCpt,ReduceLRCpt],
                                      shuffle=False))



Do you need more than the above code? Unfortunately I cannot share the data used for this.

**Comment by amahendrakar** [comment]
@AndreasPangerl,
While running the above code, I'm facing a SyntaxError stating `ModelCpt = ModelCheckpoint("model_1.h5", monitor="loss", save_best_only=True)`. Please find the gist of it [here](https://colab.research.google.com/gist/amahendrakar/be3d879f23f2a89d9cc1970fe6b917c2/39267.ipynb).

Could you please provide the completed code and a sample data to reproduce this issue. Thanks!
